src/service/overseas.entities.service.ts

```diff
@@ -58,6 +58,22 @@ export const updateOverseasEntity = async (req: Request, session: Session) => {
   logger.debugRequest(req, `Updated Overseas Entity, ${JSON.stringify(response)}`);
 };
 
+export const getCompanyRequest = async (
```

Comment:
We should have a new company.profile.ts service for getCompanyRequest. The method should be called getCompanyProfile.